### ISSUE TITLE
some problems with the predictions fo tumors

### ISSUE THREAD

**Issue by HazardFY** [zz10001/LITS2017-main1#7]
Hello! Thanks to your code!
But I have some problems with the predictions fo tumors. I only split the official train set into train/vailidation/test set. For IOU, the predictions for livers are about 0.91  but the predictions of tumors in the test set are bad( around 0.35 ), which also exits in validation set. Especially for the cases which don't have tumors or have only one small tumor,the result is around0 for tumors' predictions. 
I wonder does this situation exit in your process of training? Are the results shown in the github  tested on the official test set?

**Comment by geng123321** [comment]
Hello, may I ask if your problem has been resolved and how has it been resolved

**Comment by geng123321** [comment]
Hello, may I ask if your problem has been resolved and how has it been resolved
